.ci/install_crio.sh

```diff
@@ -148,6 +148,10 @@ sudo sed -i 's/^registries = \[/registries = \[ "docker.io"/' "$crio_config_file
 # Matches cri-o 1.12 file format
 sudo sed -i 's/^#registries = \[/registries = \[ "docker.io" \] /' "$crio_config_file"
 
+echo "Create default oci hooks dir"
```

Comment:
@Pennyzct Does this need to be created beforehand. I would have assumed that crio would have created this directory if it is not present. If this is not handled by crio, I would check with them for this behavior.

Reply:
hi~ @amshinde 
you're right~~~thanks!!!! 
After a few moments searching, I've found, that, 15 days ago, they've raised another PR https://github.com/cri-o/cri-o/pull/3203 to skip invalid default hooks directories.
I'll update cri-o version to this recent commit and drop this commit. ;).